### DISCUSSION TITLE
startsWith('directus_') - could we get rid of it?

### DISCUSSION THREAD

**Discussion by SeanDylanGoff** [directus/directus#10369] [Feature Requests]
I noticed that in quite a few places (54 to be precise) I found code similar to this:

```

const endpoint = relatedCollection.value.collection.startsWith('directus_')
	? `/${relatedCollection.value.collection.substring(9)}`
	: `/items/${relatedCollection.value.collection}`;

```

Which made me question, why are the system tables not exposed at /items/directus_files etc?
I know there are special methods (like upload) that would break consistency if they were available at /items. I understand that this is undesirable. However I don't see why the raw tables are not exposed, which would render all these distinctions obsolete.

The only downside i could have thought of are name collisions - but these are irrelevant as no other table can be called directus_files anyways, and the prefix is not user configurable either (which is absolutely an OK limitation to have)

**Comment by rijkvanzanten** [comment]
We can't get rid of it. Most system collections have some special magic going on under the hood, which requires them to have dedicated controllers and services (for example, /notifications sends emails, /fields merges with schema data, etc etc). I'd prefer having dedicated logic and endpoints with its own documentation over having a single endpoint that has different IO based on the collection you're targeting

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
